specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-11-01/examples/ListExtensions.json

```diff
@@ -0,0 +1,85 @@
+{
+  "parameters": {
+    "subscriptionId": "subId1",
+    "resourceGroupName": "rg1",
+    "clusterRp": "Microsoft.Kubernetes",
+    "clusterResourceName": "connectedClusters",
+    "api-version": "2022-11-01",
+    "clusterName": "clusterName1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subId1/resourceGroups/rg1/providers/Microsoft.Kubernetes/connectedClusters/clusterName1/providers/Microsoft.KubernetesConfiguration/extensions/ClusterMonitor",
+            "type": "Microsoft.KubernetesConfiguration/extensions",
+            "name": "ClusterMonitor",
+            "systemData": {
+              "createdBy": "string",
+              "createdByType": "Application",
+              "createdAt": "2021-09-08T05:10:57.027Z",
+              "lastModifiedBy": "string",
+              "lastModifiedByType": "Application",
+              "lastModifiedAt": "2021-09-08T05:10:57.027Z"
+            },
+            "plan": {
+              "name": "azure-vote-standard",
+              "publisher": "Microsoft",
+              "product": "azure-vote-standard-offer-id"
+            },
+            "properties": {
+              "extensionType": "azuremonitor-containers",
+              "autoUpgradeMinorVersion": false,
+              "releaseTrain": "Preview",
+              "version": "0.1.4",
+              "scope": {
+                "cluster": {
+                  "releaseNamespace": "kube-system"
+                }
+              },
+              "configurationSettings": {
+                "omsagent.secret.wsid": "a38cef99-5a89-52ed-b6db-22095c23664b",
+                "omsagent.env.clusterName": "clusterName1"
+              },
+              "provisioningState": "Succeeded",
+              "installedVersion": "0.1.4",
+              "statuses": [],
+              "isSystemExtension": false
+            }
```

Comment:
Please add new currentVersion property in all applicable places.

Reply:
@nikkomiu